AUTHORS.txt

```diff
@@ -102,6 +102,7 @@ Karneck <dschultz26@hotmail.com>
 Kavinsky <nmunozfernandez@gmail.com>
 Keithen <Keithen.Neu@gmail.com>
 Kllrt <kllrtik@gmail.com>
+KokaKolaA3
```

Comment:
Do we have his permission?

Reply:
When i stated the mod version back in the day I asked him if he would be okay to add his animations to ACE. I remember him having a talk with someone about licensing as well. Last I talked to him about it was on 1st April 2019 (what an unfortunate date). Asked him if he is still alright with them being in ace and them being renamed with the `ace_*` prefix

Conversation about it over discord, sadly in german
![image](https://user-images.githubusercontent.com/8259498/71895620-d0001600-3151-11ea-9d3e-ff25afa2f6f5.png)

Without his OK I would have not made this pr

Reply:
Das ist doch super :)